docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc

```diff
@@ -7,1458 +7,3 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 include::_attributes.adoc[]
 :categories: security
 :summary: This guide demonstrates how to use the OpenID Connect Extension to protect your web application based on the Authorization Code Flow using Quarkus.
```

Comment:
I'm not sure I understand what is the role of this document now, it is empty. I thought it could make sense to actually retain the introduction with the diagram here and then link to 1) tutorial 2) reference material

Reply:
OK, I see it has been removed